sn_protocol/src/test_utils.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use std::fmt;
+
+pub fn assert_lists<I, J, K>(a: I, b: J)
```

Comment:
can you elaborate or add docs here?

Reply:
Added docs to that function.

Reply:
maybe `assert_list_eq` would be a better name?